changes/ce/feat-11721.en.md

```diff
@@ -1,5 +1,22 @@
-Port two TLS handshake validation features from emqx 4.4
+Enhance TLS listener to support more flexible TLS verifications.
 
 - partial_chain support
+
+  If the option `partial_chain` is set to `true`, allow connections with incomplete certificate chains.
+  
+  Check the description in emqx schema for more. 
```

Comment:
Schema presented to user as "configuration manual document".
in this format: https://docs.emqx.com/en/enterprise/v5.6.1/hocon/
```suggestion
  Check the configuration manual docment for more details.
```